docs/source/How_to_install_liminal_in_airflow_on_kubernetes.md

```diff
@@ -0,0 +1,160 @@
+* [Get Started](#Get-Started)
+   * [Scripted Installation](#scripted-installation)
+   * [Manual Installation](#manual-installation)
+* [References and other resources](#references-and-other-resources)
+
+## Workflow
+Liminal deploys into the Airflow components and the Yamls home folder is mounted by EFS.
+The process of liminal is emphasized on the following diagram:
+
+![](assets/liminal-diagram.png)
+
+## Prerequisites
+Before you begin, ensure you have met the following requirements:
+
+* You have a Kubernetes cluster running with [Helm][homebrew-helm] (and Tiller if using Helm v2.x) installed
+* You have the kubectl command line [(kubectl CLI)][homebrew-kubectl] installed
+* You have the current [context][cluster-access-kubeconfig] in kubernetes' kubeconfig file
+* You have [Airflow on Kubernetes with AWS EFS][airflowInstallation]
+* You have already created [Liminal Yaml file][liminal-yaml-file] following the liminal [Getting Started Documentation][liminalGetStarted-doc]
+* Make sure that the [example repository][liminal-getting-started-project] is your workspace
+
+### Supported Distributions
+
+|Distribution | Versions |
+|-|-|
+|[Airflow][airflowImage] | apache/airflow:1.10.12-python3.6 |
+|[Airflow Helm Chart][airflowChart] | 7.14.3 |
+
+#### You may need to add the Repo
+
+```sh
+helm repo add airflow-stable https://airflow-helm.github.io/charts
+helm repo update
+```
+
+## Get Started
+
+### Scripted Installation
+
+[Use `install_liminal_in_airflow_on_kubernetes.sh`][liminal-installation-script] to install liminal in Airflow.
+* Move to the [example repository project][liminal-getting-started-project]
+* Copy the [script][liminal-installation-script] to the [example repository project][liminal-getting-started-project] and run it
+
+![](assets/install_liminal_in_airflow_on_kubernetes.gif)
+
+### Manual Installation
+
+Here are a couple of steps of how to install liminal in Airflow on Kubernetes:
+* [Find Airflow components](#Find-Airflow-pod-names)
+* [Install liminal in the components](#Install-liminal-in-Airflow-pods)
+* [Setting up liminal](#Setting-up-liminal)
+
+#### Find Airflow pod names:
+
+```sh
+echo -n "Please enter a namespace: "
+read namespace
+components=("web" "scheduler" "worker")
+namespace=${namespace}
+podNames=()
+for component in ${components[@]}
+do
+        podNames+="$(kubectl get pod -n ${namespace} -l "app=airflow,component=${component}" --no-headers -o custom-columns=":metadata.name")\n"
+
+done
+
+echo -e "The following Airflow pods are:\n${podNames}"
+```
+#### Install liminal in Airflow pods:
+
+```sh
+IFS=$'\n'|''; set -f; podNames=( ${podNames} )
+webPodName=${podNames[${#podNames[@]} - 1]}
+
+for podName in `echo -e ${podNames[@]}`
+do
+        echo "Installing liminal in pod: ${podName}"
+        kubectl exec -it -n ${namespace} ${podName} -- bash -c 'pip install --user apache-liminal'
+done
+```
+
+#### Setting up liminal:
+There are a couple of ways to deploy Yamls in Airflow.
```

Comment:
I suggest to repeat the message:

Now that Airflow contains the liminal package, we need to deploy the actual Yamls (DAGs) into airflow - by placing the Yaml files in each components` /opt/airflow/dags folder.

There are multiple ways to achieve this.

Our recommended approach is to use EFS as a shared folder containing the Liminal Yamls.
To this end, you will need to perform the deployment from a (separate) linux box, which can get hold of the Liminal Yamls and deploy them into the EFS - which we shall name "the deployment box".

One time setup:
1. Create an EFS file system
2. Mount the EFS file system onto a deployment box
```sudo mount -t nfs4 -o nfsvers=4.1,rsize=1048576,wsize=1048576,hard,timeo=600,retrans=2,noresvport <EFS_ID>:/ /mnt/efs```
2. Create a folder on an EFS drive. This will be our target for deployment 
```mkdir -p /mnt/efs/opt/airflow/liminal_home```
3. set
```export LIMINAL_HOME=/mnt/efs/opt/airflow/liminal_home```
4. Follow the guide <<<<add a good link>>>> for deploying Airflow on K8S, with EFS mount.
Point the pods' PV to the right EFS folder, like so:
```
volumeMounts:
                - name: efs-data
                  mountPath: /opt/airflow/dags
```
where efs-data is a PVC pointing to ```<EFS_ID>:/opt/airflow/liminal_home```


Once this setup is done (one time), you have:
1.  A deployment box which can deploy Liminal to an EFS folder
2. Airflow pods which will pick up the Yamls from that folder automatically.

Each time you want to deploy the Liminal Yamls from the deployment box: 
```liminal deploy <<<path to liminal user code>>>```

Reply:
Ok I see.
I will create a short video how I did it.
And we need a doc of how to create Airflow on k8s with EFS.